pkg/io/csvio/sg.go

```diff
@@ -20,14 +20,18 @@ func (w *Writer) WriteSG(collection *ir.SGCollection, vpc string) error {
 	if err := w.w.WriteAll(sgHeader()); err != nil {
 		return err
 	}
-	for _, sgName := range collection.SortedSGNames(vpc) {
-		vpcName := ir.VpcFromScopedResource(string(sgName))
-		sgTable, err := makeSGTable(collection.SGs[vpcName][sgName], sgName)
-		if err != nil {
-			return err
+	for _, vpcName := range collection.VpcNames() {
```

Comment:
I'm not sure I understand why this change is needed.
If sgName always includes vpcName, don't we get the same order before and after this change?

Reply:
In optimization mode SGs names are given
==> we cannot extract the VPC names
==> we are not able to extract the SG itself (`*ir.SG`)

Reply:
I still don't get it. We do get in the config file a list of SG objects, where each SG has a reference to its parent VPC. Can't we take it from there?
BTW, we should probably allow users to specify the SG to optimize as `<vpc name>/<sg name>`, for cases where two VPCs have SGs with the same name.

Reply:
fixed